README.md

```diff
@@ -125,7 +140,14 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+genie = Genie.create(name: "Genie")
+lamp = Lamp.create(wishes_remaining: 3)
+genie.lamp_id = lamp_id
+lamp.update(wishes_remaining: 1)
+jafar = Genie.creat(name: "Jafar")
+jafar.lamp.create(wishes_remaining: 3)
+genie.update(wishes_remaining: nil)
```

Comment:
So genie is of class Genie, and thus doesn't have a `wishes_remaining` property. That's only for lamps!

What we were looking for here is `genie.lamp = nil`